src/catch2/catch_message.hpp

```diff
@@ -110,28 +110,32 @@ namespace Catch {
     Catch::getResultCapture().emplaceUnscopedMessage( Catch::MessageBuilder( macroName##_catch_sr, CATCH_INTERNAL_LINEINFO, Catch::ResultWas::Info ) << log )
 
 
-#if defined(CATCH_CONFIG_PREFIX_ALL) && !defined(CATCH_CONFIG_DISABLE)
+#if defined(CATCH_CONFIG_PREFIX_ALL) && !defined(CATCH_CONFIG_PREFIX_MESSAGES)
+#define CATCH_CONFIG_PREFIX_MESSAGES
+#endif
```

Comment:
Like other new `CATCH_CONFIG_*` options, this should go into its own header, `src/catch2/internal/catch_config_prefix_messages.hpp`.

Reply:
Done.